django_project/feti/tests/test_scraping.py

```diff
@@ -0,0 +1,34 @@
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
+__date__ = '30/09/16'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+from django.core.management import call_command
+from django.test import TestCase
+from feti.models.campus import Campus
+from feti.models.course import Course
+from feti.models.occupation import Occupation
+from feti.models.provider import Provider
+
+
+class TestScraping(TestCase):
+    def get_campus_count(self):
+        args = []
+        opts = {}
+        call_command('scraping_campus', *args, **opts)
```

Comment:
ah, maybe i will just scrap 1 page instead make dummy data
because if i make dummy data, i will give just optimal data, that will pass